pkg/kubectl/cmd/top/top_pod_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package cmd
+package top
```

Comment:
If these files are now in a `top` directory, you should be able to remove the `top_` prefix on the filenames.